{% from 'layout/carousel.html' import carousel_navigation with context %} 
{% extends "layout/layout.html" %} 
{% set add_sidebar = true %} 
{% block article_content %} 
{{ subheading('Goals and Purpose') }}
<div>
  <p>
    The main goal of the wet lab team is to test and validate, <i>in vitro</i>, the AI-generated plasmid parts the dry lab team
    creates. Validating each individual plasmid part allows us to determine with more granularity which specific
    AI-plasmid components are functional.
  </p>
  <p>
    To achieve this goal, we utilize a pipeline in which we assemble the AI-parts into a known plasmid backbone using
    Golden Gate Assembly (GGA – a high-throughput transgene insertion method), and test the viability of the AI-parts
    using bacterial growth assays. Finally, we sequence the new working plasmid parts to confirm successful propagation
    of the AI-parts, such as the origin of vegetative replication (<i>oriV</i>) and antibiotic resistance genes (ARGs). These
    developments provide a proof-of-concept for the tractability of synthesizing plasmids de novo using dry lab’s
    machine learning framework, which sets the path forward to ultimately create fine-tuned plasmids to tackle
    antibiotic resistance.
  </p>
  <p>
    The wet lab pipeline for validating various AI-plasmid parts, in particular the <i>oriV</i> and ARGs, is described in
    detail in the following sections. Thus far, the wet lab team has synthesized and tested novel <i>oriV</i>s generated by dry
    lab, of which samples 741, 9371, 5276, 5727, and 4089 have been determined as functional.
  </p>
</div>
{{ subheading('Design') }} 
{{ subsubheading('1. Backbone and Controls') }}
<div>
  <p>
    The main AI-plasmid part the wet lab team tested was the <i>oriV</i>. In general, the AI-generated <i>oriV</i>s consisted of
    various control elements (such as RNAI-, RNAII-, rep- and rop-coding regions) and non-coding regions. As such, to
    test this part, the wet lab team cloned the dry lab <i>oriV</i>s into suitable backbones of which have similar control
    elements for their <i>oriV</i>s. This ensured compatibility between the dry lab <i>oriV</i> and backbone.
  </p>
  <p>
    A backbone conferring resistance to a specific antibiotic serves as an effective positive and negative control for
    growth assays that are essential for ascertaining dry lab <i>oriV</i> functionality. Such a backbone eliminates confounding
    variables that may skew the interpretation of experimental results by observing E. <i>coli</i> growth as well as number of
    colonies to determine if the backbone was successfully annealed to the ori, and successfully uptaken into the E.
    coli.
  </p>
  <p>
    Initially, the plasmid pSB1C3 was used as a backbone. pSB1C3 is a common plasmid backbone from E. <i>coli</i> which confers
    chloramphenicol resistance. Thus, pSB1C3 was to be used as a positive control for the experiments, since it was
    predicted that it would successfully grow on the plates. pSB1C3 was miniprepped and underwent PCR to increase the
    amount of this plasmid, transformed into E. <i>coli</i>, which was then plated. However, no growth was observed from the
    transformed E. <i>coli</i> on the chloramphenicol plates.
  </p>
  <p>
    After we repeated this experiment a number of times, the team decided to have the plasmid sequenced to check if the
    structure was correct. Sequencing results revealed that the backbone contained additional BsaI sites than initially
    described, meaning that it was being cut in additional locations, meaning the ori and the backbone would fail to be
    annealed together.
  </p>
  <p>
    This meant that afterward, a different positive control had to be used for the experiments. Interlab_NegC was used,
    however, there it exhibited a similar problem as pSB1C3. After no growth was observed again, Interlab_NegC was taken
    for sequencing and found to have an extra BsaI site, which caused the backbone to be cleaved during GGA as our GGA
    workflow utilized the exact same restriction sites.
  </p>
  <p>
    Finally, we found that eforRed/eforCP Biobrick K592012, an ampicillin (Amp) backbone was successful as a positive
    control backbone, indicated by numerous colonies on Amp agar plates. This led us to use this backbone for future
    experiments. eforRed/eforCP Biobrick K592012 had a length of 2807 bp, and contained a 861 bp long bla gene, coding
    for beta-lactamase and thus conferring Amp antibiotic resistance.
  </p>
  <p>
    <b><u>Positive Control</u></b><br>
    The positive control served to validate the wet lab team’s experimental approach of cloning dry lab <i>oriV</i>s into
    functional backbones to test their functionality. Here, this control is carried out by cloning a known working <i>oriV</i>
    (pJump27) into eforRed/eforCP Biobrick K592012 using GGA. This would theoretically create a construct that confers
    Amp resistance provided that the cloned <i>oriV</i> is functional.
  </p>
  <p>
    As such, this construct is then transformed into competent E. <i>coli</i> K12 thawed on ice. The culture was then plated on
    Amp plates, incubated at 37°C overnight, and then observed for growth of colonies the following morning (which would
    indicate successful <i>oriV</i> functionality).
  </p>
  <p>
    Once we verified that this assembled plasmid was functional by plating transformed bacteria, we began our assemblies
    using the Dry Lab-generated sequences.
  </p>
  <p>
    <b><u>Negative Control</u></b><br>
    The negative controls are made by transforming competent E. <i>coli</i> cells with a functional plasmid lacking an <i>oriV</i>
    sequence. The negative control contains only a linear plasmid backbone without an ori, since it has not been cloned
    and annealed to anything. When assembled and plated onto Amp plates, these cells will theoretically die as they have
    no <i>oriV</i> to propagate any Amp-resistance cassettes.
  </p>
  <p>
    The negative control is used to ensure the assembly’s procedure was correct, as E. <i>coli</i> without the ori to replicate
    the plasmid would not survive on the antibiotic-plated media. For our GGA using the Interlab plasmid, Interlab_NegC
    with water was used as a negative control.
  </p>
</div>
{{ subsubheading('2. Cell Preparation') }}
<div>
  <p>
    <b><u>LB Amp agar plates</u></b><br>
    Before beginning transformations, it is important to prepare nutrient rich plates with the correct antibiotic
    conditions to ensure only E. <i>coli</i> DH5ɑ that have successfully taken up the designed plasmid with antibiotic
    resistance (i.e. successfully transformed) can grow. DH5ɑ competent E. <i>coli</i> cells were used for the transformation.
    LB agar miller was dissolved in milliQ water, then autoclaved using the wet cycle. Once cooled, Amp was added to the
    agar at a concentration of 100 µg/mL. The agar was then poured into sterile petri dishes and allowed to solidify at
    room temperature, with aseptic technique applied throughout. The plates were then stored at 4°C and warmed up to
    37°C before the transformed E. <i>coli</i> cells were plated. Prior to plating of transformed E. <i>coli</i>coli, the plates were warmed
    up to room temperature to facilitate their growth.
  </p>
  <p>
    <b><u>Competent Cells</u></b><br>
    In our experiment, we used E <i>coli</i> DH5α cells due to their high efficiency in transformation and stable amplification
    {{ cite('Kostylev2015') }}. These cells were stored at -80°C and subsequently thawed on ice before use. To initiate
    the transformation, we added the assembled plasmid, containing the <i>oriV</i> drylab generated sequence with the eforCP
    backbone, to the thawed cells and incubated the mixture. We then performed a heat shock by incubating the cells at
    42°C for exactly 30 seconds to facilitate the uptake of the plasmid DNA. Following heat shock, we added SOC medium
    to the cells and then incubated the mixture at 37°C to ensure sufficient recovery. Finally, we plated the
    transformed cells on agar plates containing Amp to select for successful transformants.
  </p>
</div>
{{ subsubheading('3. Assembly') }}
<div>
  <p>
    GGA is a one-pot molecular cloning technique that allows for the combination of multiple inserts into a backbone
    without the need for multiple restriction enzymes {{ cite('Engler2008') }}(Figure 1). We used GGA to assemble all 
    our plasmids for testing of <i>oriV</i> sequences and ARGs.
  </p>
  <div>
    {{ img(new_cdn("wetlab/gga-figure-1.webp"), alt="") }}
    <p>
      <b>Figure 1.</b> Golden Gate Assembly uses type IIS restriction enzymes, which have separate recognition (yellow)
      and cut sites (green, purple, light pink). This allows for the creation of many possible unique overhangs without
      the need for multiple restriction enzymes. GGA can then proceed in a one-pot reaction, allowing for a backbone and
      multiple inserts to be combined simultaneously. Figure created with BioRender.com.
    </p>
  </div>
  <p>
    We developed a molecular cloning (MoClo) system for our <i>oriV</i> and ARG assemblies (Figure 2). Inserts for both
    experiments were assembled into backbones with a selection marker, the ARG backbone also containing an <i>oriV</i> sequence
    (Figures 4-5). Dry lab generated <i>oriV</i> sequences were either well aligned to the <i>oriV</i> control element RNAII or the
    noncoding and Rop sequence, so 5 different assemblies were used to test their functionality (Figure 3).
  </p>
  <div>
    {{ img(new_cdn("wetlab/moclo-system-figure-2.webp"), alt="") }}
    <p>
      <b>Figure 2.</b> OriV and ARG MoClo system overhangs and their assigned letters. Overhang locations in assemblies
      are indicated in Figures 3 and 4. Figure created with BioRender.com.
    </p>
  </div>
  <div>
    {{ img(new_cdn("wetlab/type-b-oriv-assembly-figure-3.webp"), alt="") }}
    <p>
      <b>Figure 3.</b> Type B <i>oriV</i> and sequences used to test generated <i>oriV</i> alignments. Dry lab has been generating
      type B <i>oriV</i> sequences, which contain sequences for RNAI, RNAII, Rop, and a noncoding region. Generated <i>oriV</i>
      sequences have been aligning best to either RNAII or the noncoding region and Rop sequence, so we made assemblies
      to test these generated sequences by themselves and with the sequence that they are best aligned to from a wild
      type (WT) <i>oriV</i> sequence. Figure created with BioRender.com.
    </p>
  </div>
  <div>
    {{ img(new_cdn("wetlab/oriv-assembly-maps-figure-4.webp"), alt="") }}
    <p>
      <b>Figure 4.</b> The assemblies used to test generated <i>oriV</i> functionality, letters indicating the locations of
      specific MoClo overhangs. Each generated <i>oriV</i> sequence was tested by itself and with a portion of a WT <i>oriV</i>
      containing the region of the <i>oriV</i> sequence that the generated sequence wasn’t best aligned to. The entire WT <i>oriV</i>
      was also used as a control. Amp was chosen as the selection marker, and bla was used for the backbone’s ARG.
      Figure created with BioRender.com.
    </p>
  </div>
  <div>
    <div style="margin: auto; width: 50%; height: auto;">
    {{ img(new_cdn("wetlab/arg-assembly-map-figure-5.webp"), alt="") }}
    </div>
    <p>
      <b>Figure 5.</b> The assembly used to test generated ARG functionality, letters indicating the locations of
      specific MoClo overhangs. A selection marker was included to allow for selection if the generated ARG was
      nonfunctional. Figure created with BioRender.com.
    </p>
  </div>
</div>
{{ subsubheading('4. Transformation') }}
<div>
  <p>
    After finishing the assemblies, we transformed our backbone eforRed/eforCP Biobrick K592012 with different dry lab
    generated <i>oriV</i> sequences. We use competent E. <i>coli</i> DH5a cells to transform the plasmids in. We add 2 uL (1 pg – 100
    ng) of our vectors to thawed competent cells. We then incubate on ice and heat shock the cells, which is followed by
    cell recovery using SOC media for 1 hour. Afterwards, we plate the liquid cell culture on Agar plates with
    corresponding antibiotics.
  </p>
  <p>
    To test the function of the ARGs, we will also transform the assembled plasmids into competent E. <i>coli</i> DH5a cells.
    We will be using double antibiotic plates, which includes CmR, our antibiotic selection marker, and Amp.
  </p>
</div>
{{ subheading('Experiments') }} 
{{ subsubheading('Preliminary experiments (Controls)') }}
<div>
  <p>
    We began with validating our positive control experiment using the pSB1C3 device and the <i>oriV</i> from the pJUMP27-1A
    plasmid from the 2024 iGEM Distribution Kit Plate. However, this vector was found to have unreported BsaI cut sites.
    Therefore, we switched to the Interlab_NegC device, but that was also discovered to also have unreported BsaI cut
    sites. Finally, we used the eforRed_eforCP K592012 device and obtained visible colonies and a valid sequencing
    result, thus verifying our positive control.
  </p>
</div>
{{ subsubheading('Main experiments (Validation)') }}
<div>
  <p>
    Next, we began assembling the AI-generated sequences generated by our dry lab team into the eforRed/eforCP K592012
    device (BBa_J435099). The first batch of sequences generated by dry lab were not found to be functional as the
    transformed bacteria did not grow into visible colonies. 6 generated <i>oriV</i>s from the second batch of sequences
    generated visible colonies, and 5 of them were confirmed to be functional via whole plasmid sequencing. Testing for
    the third batch of sequences, which includes sequences encoding for ARGs and <i>oriV</i>s, is currently in progress.
  </p>
</div>
{{ subsubheading('Calendar') }}
<div x-data="{ galleryIndex: 0 }">
  <div x-show="galleryIndex === 0" class="w-full p-8">
    {{ img(new_cdn("wetlab/experiment-calendar-may.webp"), alt="") }}
  </div>
  <div x-show="galleryIndex === 1" class="w-full p-8 break-words">
    {{ img(new_cdn("wetlab/experiment-calendar-june.webp"), alt="") }}
  </div>
  <div x-show="galleryIndex === 2" class="w-full p-8 break-words">
    {{ img(new_cdn("wetlab/experiment-calendar-july.webp"), alt="") }}
  </div>
  <div x-show="galleryIndex === 3" class="w-full p-8 break-words">
    {{ img(new_cdn("wetlab/experiment-calendar-august.webp"), alt="") }}
  </div>
  <div x-show="galleryIndex === 4" class="w-full p-8 break-words">
    {{ img(new_cdn("wetlab/experiment-calendar-september.webp"), alt="") }}
  </div>
  {{ carousel_navigation("galleryIndex", 5) }}
</div>
{{ subheading('Results') }}
<div>
 {{ subsubheading('Controls') }}
  <p>
    We first began by assembling and validating our positive and negative controls. After switching over to the
    eforRed_eforCP vector, we amplified the positive control pJUMP27 insert and verified via agarose gel electrophoresis
    its correct size of 1.6 kb before purifying, transforming, and plating.
  </p>
  <div style="margin: auto; width: 20%; height: auto;">
    {{ img(new_cdn("wetlab/pjump27-gel.webp"), alt="") }}
  </div>
  <p>
 {{ subsubheading('Batch 1') }}
  </p>
  <p>
    Unfortunately, numerous attempts at transforming the first batch of AI-generated sequences did not produce viable
    colonies on the plates. Since our positive control produced colonies, we concluded that this first batch was
    nonfunctional.
  </p>
  <div style="margin: auto">{{ img(new_cdn("wetlab/batch-1-controls.webp"), alt="") }}</div>
  <div>{{ img(new_cdn("wetlab/batch-1-results.webp"), alt="") }}</div>
  <p>
    The wet lab pipeline for validating various AI-plasmid parts, in particular the <i>oriV</i> and antibiotic resistance genes
    (ARGs), is described in detail in the following sections. Thus far, the wet lab team has synthesized and tested
    novel <i>oriV</i>s generated by dry lab, of which samples 741, 9371, 5276, 5727, and 4089 have been determined as
    functional.
  </p>
  <p>
 {{ subsubheading('Batch 2') }}
  </p>
  <p>
    After completing the required workflow to prepare the inserts, we introduced our second batch of plasmids into E.
    <i>coli</i> for uptake. This batch included eight new AI-generated <i>oriV</i> sequences, six of which produced viable colonies on
    the plates. This result aligned with our expectations, as Batch 2 showed improved alignment with the <i>oriV</i> control
    element RNAII.
  </p>
  <div>{{ img(new_cdn("wetlab/batch-2-plates.webp"), alt="") }}</div>
  <p>
    We also tested the wild-type <i>oriV</i> which the sequences aligned to, pColE, and saw viable colonies, which provides a
    reliable point of comparison for the AI-generated <i>oriV</i>s. The positive control, pJUMP27, also produced viable
    colonies, indicating that <i>oriV</i>s are indeed modular. Upon cloning this batch, we found that sequences with the IDs
    741, 4089, 5276, 5727, 6769, and 9371 produced visible colonies. Upon sequencing, we confirmed that the sequences
    with the ID 741, 4089, 5276, 5727, 9371, and the wild-type pColE1 <i>oriV</i> were indeed functional. Unexpectedly, sample
    6769 from this batch lacked a promoter but colonies were still observed. However, the sequencing data for 6769 was
    unreadable, so we cannot draw any conclusions about its functionality. The observed colonies might very well be due
    to contamination or high background levels. Upon reviewing our designs, we realized that samples lacking a promoter
    (like 6769) could theoretically produce functional plasmids if driven by the promoter of the ARG. We had assembled
    the resistance gene and <i>oriV</i> in the same orientation, which could allow the ARG's promoter to inadvertently drive
    the expression of <i>oriV</i>. To address this issue, we plan to assemble the parts in our next batch such that they run in
    opposite directions, which is currently ongoing.
  </p>
</div>
{{ subheading('Parts') }}
<div>
  <p>All experiments were carried out using the Invitrogen One Shot™ MAX Efficiency™ DH5α-T1R Competent Cells.</p>
  {{ subsubheading('Table 1. Vectors and synthesized DNA used for the cloning process') }}
  <div class="w-full overflow-auto">
    <table class="entrepreneurship">
      <tr>
        <th>Part Name</th>
        <th>Part Number</th>
        <th>Type</th>
        <th>Description</th>
      </tr>
      <tr>
        <td></td>
        <td>BBa_J435099</td>
        <td>PCR Product</td>
        <td>
          eforRed/eforCP Biobrick K592012 underwent PCR with the bla_part_B_F and bla_part_A_R primers to create a
          testing vector consisting of the Ampicillin resistance gene (bla) and its promoter, RBS, and terminator. BsaI
          cut sites and GAGT/CCTT overhangs were added in via PCR.
        </td>
      </tr>
      <tr>
        <td>eforRed/eforCP Biobrick K592012</td>
        <td>BBa_J428326</td>
        <td>PCR Product</td>
        <td>
          This consists of the pJUMP27-1A (sfGFP) <i>oriV</i>, and it is a E. <i>coli</i> plasmid origin of replication.<br>
          pJUMP27-1A (sfGFP) underwent PCR with the pJUMP27-1A(sfGFP) 2054-2075 F1 and pJUMP27-1A(sfGFP) 2633-2654 R2
          primers. BsaI cut sites and AAGG/ACTC overhangs were added in via PCR.
        </td>
      </tr>
      <tr>
        <td>pColE1_Ori</td>
        <td>BBa_K5230001</td>
        <td>WT <i>oriV</i></td>
        <td>
          Wild-type origin of replication from the plasmid pColE1. Batch 1 sequences aligned with this <i>oriV</i> and it
          functioned as a control to confirm the modularity of <i>oriV</i>s.
        </td>
      </tr>
      <tr>
        <td>4089</td>
        <td>BBa_K5230005</td>
        <td>AI <i>oriV</i></td>
        <td>
          AI-generated origin of replication by a tokenizer, named sample 4089. Has a length of 975 bp. It was aligned
          with the Type B wild-type plasmid ori from E. <i>coli</i> plasmid pT108.
        </td>
      </tr>
      <tr>
        <td>5276</td>
        <td>BBa_K5230006</td>
        <td>AI <i>oriV</i></td>
        <td>
          AI-generated origin of replication by a tokenizer, named sample 5276. Has a length of 963 bp. BBa_K5230006 was
          matched with the Type B wild-type plasmid ori from E. <i>coli</i> plasmid pT108.
        </td>
      </tr>
      <tr>
        <td>5727</td>
        <td>BBa_K5230007</td>
        <td>AI <i>oriV</i></td>
        <td>
          AI-generated origin of replication by a tokenizer, named sample 5727. Has a length of 965 bp. BBa_K5230007 was
          matched with the Type B wild-type plasmid ori from E. <i>coli</i> plasmid pT108.
        </td>
      </tr>
      <tr>
        <td>9371</td>
        <td>BBa_K5230008</td>
        <td>AI <i>oriV</i></td>
        <td>
          AI-generated origin of replication by a tokenizer, named sample 9371. Has a length of 688 bp. BBa_K5230008 was
          matched with the Type B wild-type plasmid ori from E. <i>coli</i> plasmid pT108.
        </td>
      </tr>
      <tr>
        <td>741</td>
        <td>BBa_K5230022</td>
        <td>AI <i>oriV</i></td>
        <td>
          AI-generated origin of replication by a tokenizer, named sample 741. Has a length of 827 bp. BBa_K5230022 was
          matched with the Type B wild-type plasmid ori from E. <i>coli</i> plasmid pT108.
        </td>
      </tr>
    </table>
  </div>
  {{ subsubheading('Table 2. Primers used for cloning process') }}
  <div class="w-full overflow-auto">
    <table class="entrepreneurship">
      <tr>
        <th>Part Name</th>
        <th>Part Number</th>
        <th>Type</th>
        <th>Description</th>
      </tr>
      <tr>
        <td>bla_part_B_F</td>
        <td>BBa_K5230013</td>
        <td>Primer</td>
        <td>
          Forward primer for amplicilin resistance backbone eforRed/eforCP K592012. Primer has a length of 36, 52.78% GC
          content, and annealing temperature of 66.2°C. Contains a BsaI cut site and overhang sequence GAGT.
        </td>
      </tr>
      <tr>
        <td>bla_part_A_R</td>
        <td>BBa_K5230015</td>
        <td>Primer</td>
        <td>
          Reverse primer for the ampicillin resistance backbone eforRed_eforCP K592012. Primer has a length of 36 bp,
          50% GC content, and annealing temperature of 66.2°C. Contains a BsaI cut site and overhang sequence CCTT.
        </td>
      </tr>
      <tr>
        <td>pJUMP27-1A(sfGFP) 2054-2075 F1</td>
        <td>BBa_K5230016</td>
        <td>Primer</td>
        <td>
          Forward primer for positive control OriV pJUMP27-1A(sfGFP). Primer has a length of 36, 55.56% GC content, and
          annealing temperature of 66.4°C.
        </td>
      </tr>
      <tr>
        <td>pJUMP27-1A(sfGFP) 2054-2075 R2</td>
        <td>BBa_K5230017</td>
        <td>Primer</td>
        <td>
          Reverse primer for positive control OriV pJUMP27-1A(sfGFP). Primer has a length of 36, 61.11% GC content, and
          annealing temperature of 66.4°C.
        </td>
      </tr>
      <tr>
        <td>ori_sample_A_F</td>
        <td>BBa_K5230018</td>
        <td>Primer</td>
        <td>
          Forward primer for the batch one AI-generated OriVs. Primer has a length of 22, 59.09% GC content, and
          annealing temperature of 67.3°C.
        </td>
      </tr>
      <tr>
        <td>ori_sample_B_R</td>
        <td>BBa_K5230019</td>
        <td>Primer</td>
        <td>
          Reverse primer for the batch one AI-generated OriVs. Primer has a length of 22, 50% GC content, and annealing
          temperature of 65.8°C.
        </td>
      </tr>
      <tr>
        <td>pColE1_RNAII_A_F</td>
        <td>BBa_K5230020</td>
        <td>Primer</td>
        <td>
          Forward primer with a length of 36 base pairs, used during PCR amplification of the wild-type pColE1 ori. GC
          content is 54.5%, and annealing temperature is 66.2C.
        </td>
      </tr>
      <tr>
        <td>pColE1_RR_C_R</td>
        <td>BBa_K5230021</td>
        <td>Primer</td>
        <td>
          Reverse primer with a length of 36 base pairs, used during PCR amplification of the wild-type pColE1 ori. GC
          content is 50%, and an annealing temperature of 66.2C.
        </td>
      </tr>
    </table>
  </div>
</div>
{{ subheading('Proof of concept') }}
<div>
 {{ subsubheading('Positive Control') }}
  <p>
    The <i>oriV</i> from the pJUMP27-1A (BBa_J428326) vector was cloned into the eforRed/eforCP (BBa_J435099) backbone and
    transformed into E. <i>coli</i> DH5a, which generated visible colonies. This functioned as a positive control and confirmed
    that <i>oriV</i>s are modular.
  </p>
  <div>
  {{ img(new_cdn("wetlab/screenshot-2024-10-01-at-08-39-57.webp"), alt="") }}
  </div>
 {{ subsubheading('5 Functional Sequences') }}
  <p>
    After testing batch 1 of the AI-generated <i>oriV</i>s, we found no functional sequences as no visible colonies were seen.
    Upon testing batch 2 sequences, we found 5 functional sequences, summarized below:
  </p>
  <p>Batch 2 Sequence Testing Result Summary</p>
  <div class="w-full overflow-auto">
    <table class="entrepreneurship">
      <tr>
        <th>Sequence ID</th>
        <th style="text-align: center">306</th>
        <th style="text-align: center">741</th>
        <th style="text-align: center">1670</th>
        <th style="text-align: center">4089</th>
        <th style="text-align: center">5276</th>
        <th style="text-align: center">5727</th>
        <th style="text-align: center">6769</th>
        <th style="text-align: center">9371</th>
      </tr>
      <tr>
        <td>Visible colonies?</td>
        <td style="background-color: #ff7f7f; text-align: center;">No</td>
        <td style="background-color: #90ee90; text-align: center;">Yes</td>
        <td style="background-color: #ff7f7f; text-align: center;">No</td>
        <td style="background-color: #90ee90; text-align: center;">Yes</td>
        <td style="background-color: #90ee90; text-align: center;">Yes</td>
        <td style="background-color: #90ee90; text-align: center;">Yes</td>
        <td style="background-color: #90ee90; text-align: center;">Yes</td>
        <td style="background-color: #90ee90; text-align: center;">Yes</td>
      </tr>
      <tr>
        <td>Verified by whole plasmid sequencing?</td>
        <td style="text-align: center">N/A</td>
        <td style="background-color: #90ee90; text-align: center;">Yes</td>
        <td style="text-align: center">N/A</td>
        <td style="background-color: #90ee90; text-align: center;">Yes</td>
        <td style="background-color: #90ee90; text-align: center;">Yes</td>
        <td style="background-color: #90ee90; text-align: center;">Yes</td>
        <td style="background-color: #ff7f7f; text-align: center;">No</td>
        <td style="background-color: #90ee90; text-align: center;">Yes</td>
      </tr>
    </table>
  </div>
  <p>
    The pColE1 wild-type <i>oriV</i> was also confirmed to be functional. The following is a whole plasmid sequencing result
    showing 100% alignment between the eforRed/eforCP + 9371 sequence in silico assembly and the isolated plasmid:
  </p>
  <div>
  {{ img(new_cdn("wetlab/screenshot-2024-10-01-at-08-41-25.webp"), alt="") }}
  </div>
  <p>
    These results demonstrate our proof of concept, indicating that the AI model generated functional <i>oriV</i>s which allow
    the plasmids to replicate in vivo. Currently, we are testing batch 3 of the AI-generated sequences, which includes
    both <i>oriV</i>s and ARGs.
  </p>
</div>
{{ subheading('Safety') }} 
{{ subsubheading('Lab Spaces') }}
<div>
  <p>
    For the safety of our team members, we ensure all experiments are performed in functional, appropriate, isolated and
    properly organized facilities.
  </p>
  <p>
    All our experiments are performed in a biosafety level 1 facility, with appropriate Personal Protective Equipment
    open to access. All our laboratory facilities are labelled and documented to the drawer, with proper labeling and
    segregation mechanisms in place.
  </p>
  <p>All wetlab members are required to undergo rigorous safety training. This includes:</p>
  <ol>
    <li>
      A 6 hour online module specifying all operational risks, safety, emergency situations, and proper lab attire
      during lab work in Biozone.
    </li>
    <li>A lab tour with the BIozone lab manager to ensure knowledge about emergency equipment.</li>
    <li>Equipment training with the lab technician to ensure operational knowledge on standard operating equipment.</li>
  </ol>
</div>
{{ subsubheading('Lab Bench - Room 403 of the Wallberg facility, Biozone') }}
<div  style="margin: auto; height: auto; width: 50%">
  {{ img(new_cdn("wetlab/p1.webp"), alt="We have a designated drawer for laboratory coats. They are accessible to all
  members working in the lab, and stored separately from chemicals or apparatus to prevent contamination.") }}
</div>
<div>
  {{ img(new_cdn("wetlab/screenshot-2024-10-01-at-08-02-14.webp"), alt="All our drawers are labelled and designated for
  certain items. For example we have a drawer for stationery, sample kits, tube racks, pipettes etc. This allows neat
  inventorizing.") }}
</div>
<div style="margin: auto; height: auto; width: 50%">
  {{ img(new_cdn("wetlab/p5.webp"), alt="Old chemicals are stored separately on a rack to prevent mixup from new ones.")
  }}
</div>
<div style="margin: auto; height: auto; width: 50%">
  {{ img(new_cdn("wetlab/screenshot-2024-10-01-at-08-06-13.webp"), alt="Solid and liquid waste are discarded into
  designated black bins and white containers. Biowaste, chemical waste, contaminated glassware are all discarded
  separately into different containers for proper disposal. These waste bins are emptied when 2/3 full every other day
  to prevent spillage.") }}
</div>
<div style="margin: auto; height: auto; width: 50%">
  {{ img(new_cdn("wetlab/p7.webp"), alt="We store lighter items such as pipette tips and racks on the top shelf, to
  reduce risk of falling as well as optimising space due to their stackable nature.") }}
</div>
<div>
  {{ img(new_cdn("wetlab/screenshot-2024-10-01-at-08-07-36.webp"), alt="iGEM keeps its own reagents in this 4°C Fridge
  and -20°C Freezer. This prevents cross contamination from other lab users. All reagents are properly labelled.") }}
</div>
{{ subsubheading('WallBerg Building Room 301 - Biozone') }}
<div>
  {{ img(new_cdn("wetlab/screenshot-2024-10-01-at-08-10-20.webp"), alt="Our -80°C freezer is located here. Since Biozone
  is a shared facility, all our boxes are labelled with a sticker and a barcode. Thermal gloves are provided by the
  facility to prevent freeze-burn.") }}
</div>
<div style="margin: auto; height: auto; width: 50%">
  {{ img(new_cdn("wetlab/p12.webp"), alt="Our large centrifuge is located here. We ensure appropriate rotor settings are
  being used.") }}
</div>
<div style="margin: auto; height: auto; width: 40%">
  {{ img(new_cdn("wetlab/p13.webp"), alt="Our incubator is located here. All our plates are properly labelled before
  being put inside to prevent mix up from other lab users. ") }}
</div>
{{ subsubheading('iGEM secondary working station - Wallberg Building Room 303 - Biozone') }}
<div>
  {{ img(new_cdn("wetlab/screenshot-2024-10-01-at-08-16-17.webp"), alt="Storage Space: Our -20°C Freezer and extra 4°C
  fridge space are located in the room.") }}
</div>
<div>
  {{ img(new_cdn("wetlab/screenshot-2024-10-01-at-08-18-15.webp"), alt="To prevent confusion, all our gel reagents and
  apparatus are located in the same room space.") }}
</div>
<div style="margin: auto; height: auto; width: 50%">
  {{ img(new_cdn("wetlab/p18.webp"), alt="Our nanodrop and gel visualizer machines are also located in the room.") }}
</div>
<div style="margin: auto; height: auto; width: 30%">
  {{ img(new_cdn("wetlab/p19.webp"), alt="Biozone stores all its plastic consumables such as pipette tips, Eppendorf
  tubes, serological pipettes and plates at the back wall of the room. ") }}
</div>
<div>
  {{ img(new_cdn("wetlab/screenshot-2024-10-01-at-08-20-34.webp"), alt="The iGEM bunsen burner is located at the back of
  the room. The space is kept clear to prevent accidents. An extra gas valve is in place for safety.") }}
</div>
<div>
  {{ img(new_cdn("wetlab/screenshot-2024-10-01-at-08-21-35.webp"), alt="The room consists of 2 benchtop centrifuges,
  thermocyclers and water baths for experimental use.") }}
</div>
{{ subsubheading('Risks and Risk Management') }}
<div>
  <p>
    <b>Human Health/Safety Factors - Lab Members :</b><br />
    iGEM Toronto prides itself in involving both experienced and new researchers into collaborating to create impactful
    and meaningful synthetic biology projects. As a group of undergraduates, some of our members do not have prior wet
    lab experience. Therefore, proper training, lab safety, abiding to EHS regulations and sufficient understanding of
    experimental procedures must be implemented before experiments are allowed to proceed. Furthermore, biologicals and
    chemicals are invaluable resources. Proper coordination and inventorization measures must take place for a large
    group of students to seamlessly carry on from previous experiments without generating extra waste.
  </p>
  <p>
    We have been in touch with Endang Susilawati – the lab manager for our lab space and a member of the biosafety
    council from the University of Toronto’s Department of Chemical Engineering. WHMIS guidelines and the
    <a href="https://ehs.utoronto.ca/" target="_blank">University of Toronto’s Health and Safety committee</a> govern our work.
  </p>
  <p>
    The list of safety measures taken to ensure lab member’s individual safety as well as proper training are attached
    in the table below:
  </p>
</div>
{{ subsubheading('Human Factors - Risk') }}
<div class="w-full overflow-auto">
  <table class="entrepreneurship">
    <tr>
      <th>Risk</th>
      <th>Hazard</th>
      <th>Stakeholder(s)</th>
      <th>Safety Measures Implemented</th>
    </tr>
    <tr>
      <td>Lab members lack of knowledge on experimental safety and protocol</td>
      <td>
        Misuse of apparatus, causing apparatus breakage and more importantly damage or injury to the individual and
        surrounding team members
      </td>
      <td>Biozone, iGEM members</td>
      <td>
        <p>
          All iGEM members must complete a <b>laboratory safety course CHE Summer Training, EHS policy course </b>
          <a href="https://www.utsc.utoronto.ca/ehs/ehs-101-whmis-lab-safety" target="_blank">EHS101</a>,
          <b>EHS102 and a Biosafety course </b><a href="https://www.utsc.utoronto.ca/ehs/ehs-601-biosafety" target="_blank">EHS601</a>
          provided by the University of Toronto's Department of Chemical Engineering. A safety orientation covering lab
          access and rules, biosafety equipment and differences, aseptic technique, proper disinfection and
          sterilisation methods, emergency procedures, rules for sample transport in open and closed spaces, physical
          and personnel biosecurity, proper use of equipment and chemical, fire and electrical safety are required.
        </p>
        <p>
          All iGEM members must INDIVIDUALLY file a Research Registration Form. This form requires individuals to attach
          experimental protocol, safety measures and research the MSDS of all chemicals and biologicals that are
          involved. The form is later reviewed by the University of Toronto's Health and Safety department for approval.
          A one-on-one interview with our primary supervisor is also required where members are tested about
          experimental procedures and lab safety to ensure all members are up to speed. iGEM Toronto also provides its
          wet lab members a set of master protocol including proper waste disposal procedures, and experimental methods
          to prevent confusion.
        </p>
      </td>
    </tr>
    <tr>
      <td>Lab members inability to obtain PPE</td>
      <td>Spillage or exposure leading to injury and infection on surrounding team members and wet lab member</td>
      <td>Biozone lab users, iGEM members</td>
      <td>
        iGEM keeps its own set of PPE in a drawer in room 403 (see table for iGEM main bench). Lab coats, safety gloves,
        safety glasses and goggles are provided by biozone. Lab members are not allowed to enter a facility without
        wearing proper PPE
      </td>
    </tr>
    <tr>
      <td>Mixup and confusion between lab shifts</td>
      <td>Waste of chemicals, iGEM equipment being scattered all over the place</td>
      <td>Biozone lab users, iGEM members</td>
      <td>
        To prevent mixup and confusion between lab shifts causing excessive chemicals to be generated and reagents being
        left all over the place, all wet lab members must include the storage location and detailed lab notes for every
        shift. This includes photos of the storage space of new chemicals/biologicals and experiments conducted to
        prevent redundancy. Our Protocol Police division is implemented to check lab notes every shift to ensure members
        have taken part into proper inventory procedures.
      </td>
    </tr>
    <tr>
      <td>Improper treatment of chemical/biological waste</td>
      <td>Environmental release</td>
      <td>Neighbouring water bodies (Lake Ontario), Biozone drainage system</td>
      <td>
        The safety orientation, department provided training and RRF include proper disposal methods of all waste. All
        level 1 biologicals are required to be autoclaved before disposal down the sink. Chemical waste must be properly
        labelled with a container stipulating the type, and is stored properly until 2/3 full then sent to the
        department of Chemical Engineering's waste treatment facility to be further processed.
      </td>
    </tr>
    <tr>
      <td>Lab members inexperience in using new equipment</td>
      <td>Injury, breakage of apparatus</td>
      <td>Biozone staff, lab users, iGEM members</td>
      <td>
        To reduce inexperience and prevent misconceptions on equipment usage, our Biozone lab manager and technicians
        organized an equipment training tour to teach iGEM students how to use common equipment such as the autoclave
        and the centrifuge. Biozone also organized an orientation to reinforce lab safety and procedures. All students
        must participate in the training before allowed entry to lab facilities.
      </td>
    </tr>
    <tr>
      <td>Irresponsible use of equipment</td>
      <td>Injury, breakage, inconvenience to other lab members</td>
      <td>Biozone lab users, iGEM members</td>
      <td>
        iGEM members must show consideration and responsibility to other lab users and their needs. This includes owning
        up to mistakes and proper usage of facilities in a shared lab space such as Biozone. All iGEM members are
        required to book equipment on the Biozone Calendar portal and adhere strictly to the time frame to prevent
        interfering with other people's experiments.
      </td>
    </tr>
  </table>
</div>
{{ subsubheading('Anticipating Future Risk') }}
<div>
  <p>
    In this project, wetlab validates the dry lab sequences generated. Novel <i>oriV</i>s should not confer hazardous 
    functionality to E. <i>coli</i> DH5a cells.
  </p>
  <p>
    The main risk of our technology is potential misuse of our language model and the uncontrolled development of our
    language model generating unforeseen and potentially hazardous sequences. Currently in our experimental procedures,
    We define risk as a combination of <b>vulnerability, magnitude of hazard</b> and <b>exposure</b>, and our hardware team has been
    working on methods to reduce vulnerability and exposure.
  </p>
  <p>
    <u><b>Vulnerability</b></u><br>
    Wetlab ensures all members are dressed in appropriate personal protection equipment (PPE), and all reagents are
    contained in the lab. Handwashing procedures are mandated, and sick members are not allowed into the lab to reduce
    vulnerability.
  </p>
  <p>
    <u><b>Exposure</b></u><br>
    Our <b>hardware team</b> is working on a <b>colony counter</b> and <b>RFID chip</b> to automate colony counting and
    allow better inventorization of chemicals. In our workflows, validation of plasmids are performed through observing
    and counting colonies. An automated method of validation reduces the need of human exposure to bacteria containing
    novel sequences. The RFID chip also reduces human exposure inventorization of chemicals and provides a thorough
    tracking mechanism - allowing easier root cause identification and chemical location should errors occur. These
    processes reduce exposure, preventing potential containment breaches and consequently, risk.
  </p>
</div>

{% endblock %}
